src/routers/admin.ts

```diff
@@ -93,7 +93,7 @@ const createAdminRouter = (config: AppConfig, authFilter: (scopes: string[]) =>
           });
 
           if (!updatedFiles) {
-            res.status(400).send(`No files updated.`);
+            res.status(400).json({ status: 'Failure', message: `No files updated.` });
```

Comment:
is this `Failure` status from a response standard? hadn't seen it

Reply:
there's not a defined common response structure, this is a place for improvement. I just wanted to switch to a json response body instead of text. These are administrator functions called directly and not interacted with through another application so the response structure (at this point) is arbitrary.